src/controller/python/chip/clusters/ClusterObjects.py

```diff
@@ -254,6 +272,18 @@ class ClusterAttributeDescriptor:
 
     The implementation of this functions is quite tricky, it will create a cluster object on-the-fly, and use it for actual encode / decode routine to save lines of code.
     '''
+
+    def __init_subclass__(cls, *args, **kwargs) -> None:
+        """Register a subclass."""
+        super().__init_subclass__(*args, **kwargs)
+        try:
+            if cls.cluster_id not in ALL_ATTRIBUTES:
+                ALL_ATTRIBUTES[cls.cluster_id] = {}
+            # register this clusterattribute in the ALL_ATTRIBUTES dict for quick lookups
+            ALL_ATTRIBUTES[cls.cluster_id][cls.attribute_id] = cls
+        except NotImplementedError:
+            pass # we can safely ignore this
+        
```

Comment:
```suggestion
            pass  # we can safely ignore this

```
should pass the CI, though again, a comment explaining _why_ it's safe to ignore would be better.

Reply:
oh my, forgot the I ha this on 2 places. I'll adjust the comment.
About style/linting issues... So there actually is some style applied